pkg/logcli/client/file.go

```diff
@@ -78,6 +78,7 @@ func (f *FileClient) Query(q string, limit int, t time.Time, direction logproto.
 		direction,
 		uint32(limit),
 		nil,
+		"",
```

Comment:
What do you thinking about hinting `logcli` here? Do we already capture the user agent somewhere perhaps?

Reply:
This empty string is just to satisfy the interface.(And at the moment, I don't know if we need to accept explicity `hint` for query from LogCLI)

And we do capture User agent for logcli here
https://github.com/grafana/loki/blob/main/pkg/logcli/client/client.go#L181

Although I don't know how is it related?

Reply:
What I'm getting at is: should we use this `hint` to also inform us when different clients are used? This might come in handy one day. If we're already logging the User-Agent then we probably don't need to.

Reply:
Ah I see. So currently we plan to use `hint` for mainly feature toggles. But sure yea, it could have other use cases going forward.

And yea, we may not need to use it for User-Agent, as we already set that header for any request sent by LogCLI.

Reply:
Sure, but do we _log_ the user-agent for these requests?
Don't let this comment block the PR, but I think it might be worth considering this